src/plugins/timeline.ts

```diff
@@ -37,6 +40,7 @@ export class TimelinePlugin extends BasePlugin<TimelinePluginEvents, TimelinePlu
 
   constructor(options?: TimelinePluginOptions) {
     super(options || {})
+    defaultOptions.formatTimeCallback = this.defaultFormatTime
```

Comment:
This is a little bit odd. I would move the default function out of the class directly to where the defaultOptions are specified. Otherwise instances of the plugin essentially overwrite a global object, affecting all other instances. Practically no difference but architecturally odd.